spec.html

```diff
@@ -45020,7 +45020,7 @@ <h1>
         <h1>
           AsyncGeneratorUnwrapYieldResumption (
             _resumptionValue_: a Completion Record,
-          ): either a normal completion containing an ECMAScript language value or an abrupt completion
+          ): either a normal completion containing an ECMAScript language value or a throw completion
```

Comment:
This is wrong; it explicitly constructs and returns a return completion.

Reply:
Whoops, fixed. (My static analysis did pick up on this, but I missed it.)

Reply:
should it then say "return completion" rather than just "abrupt"?

Reply:
It can also produce a throw completion.

Reply:
and we'd prefer "abrupt" to the more precise "return completion or throw completion"?

Reply:
I prefer it, at any rate, yes.

Reply:
Personally, I'd prefer the more precise version, but I was confining this PR to cases of just "abrupt" -> "throw".